### PR TITLE
Separate footer menu into its own app

### PR DESCRIPTION
**Pull request by melzreal**

### CODE REVIEW

File: footer_menu/models.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        help_text=_("Title of submenu - leave blank if there is no custom submenu"),
+    )
+    icon = models.ForeignKey(
+        "wagtailimages.Image",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    show_when = models.CharField(
```
Comment (Rhian) [inline footer_menu/models.py:80]: unless I'm misunderstanding this, do we need this? As we don't have logged-in users?

File: footer_menu/models.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            return self.url
+        return None
+
+    @property
+    def slug_of_submenu(self):
+        if self.title_of_submenu:
+            return slugify(self.title_of_submenu)
+        return None
+
+    def show(self, authenticated):
```
Comment (Rhian) [inline footer_menu/models.py:112]: As above, do we need this without having authentication on the site?

File: footer_menu/models.py (hunk 1 above)
Comment (melzreal) [reply footer_menu/models.py:80]: maybe not, I thought it was a good idea to have the option to create multiple menus for later on - but in fact its something we might never use - will take this out!

**Review by Rhian — APPROVED** [review]
Looks good to go.

**Pull request merged**